GrandCommonDivisor.Core/Utils/GCDUtils.cs

```diff
@@ -0,0 +1,36 @@
+﻿using GrandCommonDivisor.Core.models;
+using System;
+
+namespace GrandCommonDivisor.Core
+{
+    public static class GCDUtils
+    {
+        public static int CalculateGCD(int firstN, int secondN)
+        {
+            if (secondN == 0)
+            {
+                return firstN;
+            }
+            int modulo = firstN % secondN;
+
+            return CalculateGCD(secondN, modulo);
```

Comment:
Also, try to rewrite this without recursion, because it may cause StackOverflowException (also read about what is it and why it may occur). This exception won't happen here, but I've never seen recursion in a real production code

Reply:
NOT fixed, I have to figure out how to do it, with recursion it is easy. Will probably address this tomorrow.

Reply:
Now, this one is fixed. Normal algorithm employed.